ipaserver/install/bindinstance.py

```diff
@@ -1377,10 +1379,12 @@ def uninstall(self):
 
         self.named_conflict.unmask()
 
-        certmonger.stop_tracking(certfile=paths.BIND_DNS_OVER_TLS_CRT)
-        certmonger.stop_tracking(certfile=paths.BIND_DNS_OVER_TLS_KEY)
-        services.knownservices.unbound.disable()
-        services.knownservices.unbound.stop()
+        if self.sstore.restore_state("dns_over_tls", "enabled"):
+            if not self.sstore.restore_state("dns_over_tls", "external_crt"):
+                certmonger.stop_tracking(certfile=paths.BIND_DNS_OVER_TLS_CRT)
+                certmonger.stop_tracking(certfile=paths.BIND_DNS_OVER_TLS_KEY)
```

Comment:
The stop_tracking for the cert is sufficient. You can drop the stop tracking for the key (they are a pair).

Reply:
dropped.